vaadin-grid-flow-parent/vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/FooterRow.java

```diff
@@ -19,7 +19,6 @@
 import java.util.List;
 
 import com.vaadin.flow.component.Component;
-import com.vaadin.flow.component.grid.AbstractRow.AbstractCell;
```

Comment:
Hmm....doesn't compile for me without this import:

<img width="768" alt="Screenshot 2022-08-09 at 17 08 09" src="https://user-images.githubusercontent.com/1222264/183670922-151ef720-6338-4513-9d0a-2ea2e147f8f0.png">

Reply:
Weird, let's restore them to be safe.